mobile/onied/lib/blocs/profile_info/profile_info_bloc.dart

```diff
@@ -10,7 +11,7 @@ class ProfileInfoBloc extends Bloc<ProfileInfoBlocEvent, ProfileInfoBlocState> {
   ProfileInfoBloc({required this.repository}) : super(LoadingState()) {
     on<LoadUser>(_onLoadUser);
     on<UpdateUserModel>(_onUpdateUserModel);
-    // on<SaveUserInfo>(); // TODO: Save user to backend
+    on<SaveUserInfo>(_onSaveUserInfo); // TODO: Save user to backend
```

Comment:
Вроде как todo выполнен

```suggestion
    on<SaveUserInfo>(_onSaveUserInfo);
```